flowtorch/bijectors/__init__.py

```diff
@@ -35,26 +37,28 @@
     ("Affine", Affine),
     ("AffineAutoregressive", AffineAutoregressive),
     ("AffineFixed", AffineFixed),
+    ("Fixed", Fixed),
+    ("ConvCouplingBijector", ConvCouplingBijector),
+    ("CouplingBijector", CouplingBijector),
     ("ELU", ELU),
     ("Exp", Exp),
     ("LeakyReLU", LeakyReLU),
-    ("Permute", Permute),
+    ("VolumePreserving", VolumePreserving),
```

Comment:
I'm not sure of what to do with those imports, why do they differ when I call `python scripts/generate_imports.py`?